index.d.ts

```diff
@@ -404,3 +404,40 @@ export function stringifyUrl(
 	object: UrlObject,
 	options?: StringifyOptions
 ): string;
+
+/**
+Filter query parameters from a url.
+
+@param url The url containing the query parameters to filter.
+@param includeParams The names of the query parameters to retain. All other query parameters will be removed from the url.
```

Comment:
```suggestion
@param keysToKeep The names of the query parameters to retain. All other query parameters will be removed from the URL.
```

?

I think that name might be clearer. The description needs to be updated for that too.